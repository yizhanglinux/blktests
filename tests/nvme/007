#!/bin/bash
# SPDX-License-Identifier: GPL-2.0+
# Copyright (c) 2017-2018 Western Digital Corporation or its affiliates.
#
# Test NVMeOF target creation with a file backed ns.

. tests/nvme/rc

DESCRIPTION="create an NVMeOF target with a file-backed ns"
QUICK=1

requires() {
	_nvme_requires
	_require_nvme_trtype_is_fabrics
}

test() {
	echo "Running ${TEST_NAME}"

	_setup_nvmet

	_nvmet_target_setup --blkdev file

	_nvmet_target_cleanup

	echo "Test complete"
}
