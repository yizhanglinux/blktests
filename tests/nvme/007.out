Running nvme/007
Test complete
